content/v0.0.1/index.md

```diff
@@ -6,3 +6,290 @@ aliases: ["/en/"]
+    auth token or any other secret must not be added to the entry.
+
+## Specification
+
+The key words “MUST”, “MUST NOT”, “REQUIRED”, “SHALL”, “SHALL NOT”, “SHOULD”, “SHOULD NOT”, “RECOMMENDED”, “MAY”, and
+“OPTIONAL” in this document are to be interpreted as described in [RFC 2119](https://www.ietf.org/rfc/rfc2119.txt).
+
+When the same event happens multiple times, the message must always be the same and other fields must be used as
+discriminant.
+
+1. A field for the `message` MUST be provided in each log entry.
+2. The content of a `message` field MUST be a description of an event occurrence.
+3. The content of the `message` field SHOULD be unique within your system.
+4. The content of the `message` field MUST NOT contain variable content and MUST be a static.
+5. The `message` field MAY be named differently.
+6. A field for the `timestamp` MUST be provided in each log entry.
+7. The content of the `timestamp` field MUST include year, month, day, hour, minute, second, and millisecond of the
+   event occurrence.
+8. The content of the `timestamp` field MAY include a timezone information.
+9. The `timestamp` field MAY be named differently.
+10. A field for the `severity` MUST be provided in each log entry.
+11. The `severity` field MAY be named differently.
+12. The content of the `severity` field SHOULD be one of ***debug***, ***info***, ***warning*** (or ***warn***),
+    ***error***, ***fatal***.
+13. The content of the `severity` field MUST be all lowercase or uppercase.
+14. A ***debug*** `severity` SHOULD be used only for debugging purposes.
+15. An ***info*** `severity` SHOULD ....
+16. A ***warning*** `severity` (or ***warn***) SHOULD report an event may cause problem to an application but that
+    still does not affect the user directly.
+17. An ***error*** `severity` SHOULD be used when an event needs attention
+    because it affects the user of the application in a disruptive way.
+18. A ***fatal*** `severity` SHOULD flag an entry that caused an application to crash or terminate.
+19. A field for the `scope` SHOULD be provided.
+20. The content of a `scope` field MUST be a reference to the creator of the log.
+21. The `scope` field MAY be named differently.
+22. Further entry fields SHOULD be added to a log entry to better.
+23. The log entry fields MUST NOT contain any application secret.
+24. The log entry fields MUST NOT contain any sensible customer
+    data ([PII](https://en.wikipedia.org/wiki/Personal_data)).
+25. The log entry fields MAY be encoded in different formats.
+26. It is RECOMMENDED to add tracing information to a log entry.
+27. The log entry fields SHOULD not contain duplicated fields.
+
+## Examples
+
+### A log entry with different valid fields
+
+```
+[
+  {
+    "severity": "WARN",
+    "scope": "web-app",
+    "message": "connection to database reset",
+    "timestamp": "2022-05-14T14:16:15+00:00",
+    "fields.trace_id": "438c908b-3baf-4374-84ad-eaed7b0faedd"
```

Comment:
Side-effects shouldn't be part of the standard. Most structured data formats support nesting directly.

Reply:
What do you mean by that?
Not sure I am following this one

Reply:
As far I understand for the examples,

```json
{
  "fields.trace_id": "foo"
}
```

is functionally the same as

```json
{
  "fields": {
    "trace_id": "foo"
  }
}
```

Only the latter should be supported. The former is a side-effect to get into the latter, and should not be considered.

Reply:
Changed the example :) 
So it should be confusing people